### PR TITLE
Issue #SB-0000 fix:Resolved the codesmells

### PR DESCRIPTION
**Pull request by sujithsojan**

### CODE REVIEW

**Review by subranil — APPROVED** [review]

File: src/app/profile/sub-profile-edit/sub-profile-edit.page.ts [hunk 1]
```diff
@@ -200,14 +200,13 @@ export class SubProfileEditPage {
   }
 
   handleHeaderEvents($event) {
-    switch ($event.name) {
-      case 'back':
-        this.telemetryGeneratorService.generateBackClickedTelemetry(
-          PageId.CREATE_MANAGED_USER,
-          Environment.USER,
-          true
+    if($event.name == 'back')
```
Comment (swayangjit) [inline src/app/profile/sub-profile-edit/sub-profile-edit.page.ts:203]: @sujithsojan  Please add "===" for any string comparision

File: src/app/textbook-view-more/textbook-view-more.page.ts [hunk 2]
```diff
@@ -67,10 +67,9 @@ export class TextbookViewMorePage {
   }
 
   private handleHeaderEvents(event: { name: string }) {
-    switch (event.name) {
-      case 'back':
-        this.location.back();
-        break;
+    if(event.name == 'back')
```
Comment (swayangjit) [inline src/app/textbook-view-more/textbook-view-more.page.ts:70]: @sujithsojan  Please add "===" for any string comparision

File: src/app/profile-settings/profile-settings.page.ts [hunk 3]
```diff
@@ -318,10 +318,9 @@ export class ProfileSettingsPage implements OnInit, OnDestroy, AfterViewInit {
   }
 
   handleHeaderEvents($event) {
-    switch ($event.name) {
-      case 'back':
-        this.handleBackButton(true);
-        break;
+    if($event.name == 'back')
```
Comment (swayangjit) [inline src/app/profile-settings/profile-settings.page.ts:321]: @sujithsojan  Please add "===" for any string comparision

File: src/app/notification/notification.page.ts [hunk 4]
```diff
@@ -136,10 +136,9 @@ export class NotificationPage implements OnInit {
   }
 
   private handleHeaderEvents(event) {
-    switch (event.name) {
-      case 'back':
-        this.telemetryGeneratorService.generateBackClickedTelemetry(PageId.NOTIFICATION, Environment.NOTIFICATION, true);
-        break;
+    if(event.name == 'back')
```
Comment (swayangjit) [inline src/app/notification/notification.page.ts:139]: @sujithsojan  Please add "===" for any string comparision

**Review by swayangjit — APPROVED** [review]

**Pull request merged**